lib/rpush/client/active_model/gcm/notification.rb

```diff
@@ -6,6 +6,7 @@ module Notification
           def self.included(base)
             base.instance_eval do
               validates :registration_ids, presence: true
+              validates :priority, inclusion: { in: Rpush::Client::ActiveModel::Apns::Notification::APNS_PRIORITIES }, allow_nil: true
```

Comment:
Personally, I'd not reuse the APNS priority, because while `normal` and `high` correspond to `5` and `10` in iOS, this might change. I'd rather have `GCM_PRIORITIES`.

@NeoElit, what do you think? Since you also seem to use GCM and want this gem to support more of its features.